dev/MRTCore/TestApps/winui_desktop_packaged_app/winui_desktop_packaged_app (Package)/build/Microsoft.WinUI.AppX.targets

```diff
@@ -0,0 +1,174 @@
+<!--
+  Copyright (c) Microsoft Corporation. Licensed under the MIT License
+-->
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+    
+    <!-- Replaces AppxManifest WinRT registrations for Microsoft.UI.Xaml.dll, post-GenerateAppxManifest. -->
```

Comment:
is this correct? should be something else?

Reply:
Oh, never mind, I see that this is a test app (testing winui consuming mrtcore).